hack/quick-install.sh

```diff
@@ -5,12 +5,29 @@ set -o xtrace
 
 export CERT_MANAGER_VERSION=v1.9.1
 export KNATIVE_EVENTING_VERSION=v1.14.3
+export JAEGER_OPERATOR_VERSION=v1.57.0
 
+# Production dependencies
```

Comment:
Rather then just comment the code, add a log message so people running this script also know what is going on.

```suggestion
echo "Installing production dependencies"
```

Reply:
While theoretically this make sense, practically this gets lost in the output of the script so its not useful. Also, we don't want to redirect the output to somewhere so its more clear if something goes wrong.